test/dlc_tests/sanity/test_ecr_scan.py

```diff
@@ -271,6 +271,9 @@ def test_ecr_enhanced_scan(image, ecr_client, sts_client, region):
     :param sts_client: boto3 Client for STS
     :param region: str Name of region where test is executed
     """
+    if src_utils.is_1p_owned_ecr(image):
+        LOGGER.info(f"Not scanning image {image} because it is owned by 1P team")
+        return
```

Comment:
Looking at the test job reports from [91f4247](https://github.com/aws/deep-learning-containers/pull/3585/commits/91f424709ebcfec51e9ddf0240629c9f80516d56), it seems that you'll also need to skip the test `sanity/test_safety_report_file.py::test_safety_file_exists_and_is_valid` in the same way as this.